exporter.go

```diff
@@ -402,7 +414,20 @@ func main() {
 	}
 
 	log.Printf("NGINX Prometheus Exporter has successfully started")
-	log.Fatal(srv.Serve(listener))
+	if *securedMetrics {
+		_, err = os.Stat(*sslServerCert)
+		if err != nil {
+			log.Fatalf("Cert file not set, not readable or non-existent. Make sure you set -web.ssl-server-cert when starting your exporter with -web.secured-metrics true.\n%v", err)
```

Comment:
for consistency with the other log messages when an error is printed:
```suggestion
			log.Fatalf("Cert file is not set, not readable or non-existent. Make sure you set -web.ssl-server-cert when starting your exporter with -web.secured-metrics=true: %v", err)
```